packages/react-components/react-progress/src/components/ProgressBar/ProgressBar.test.tsx

```diff
@@ -47,4 +53,27 @@ describe('ProgressBar', () => {
     expect(result.getByRole('progressbar').getAttribute('aria-valuemin')).toEqual('0');
     expect(result.getByRole('progressbar').getAttribute('aria-valuemax')).toEqual('1');
   });
+  describe('Max', () => {
+    it('gives the proper error message when max is negative', () => {
+      const max = -1;
+      const errorMsg = `The prop 'max' must be greater than 0. Received max: ${max}`;
+      render(<ProgressBar max={max} />);
+      expect(console.error).toHaveBeenCalledWith(errorMsg);
+    });
+    it('gives the proper error message when max is zero', () => {
```

Comment:
I think we should add a test for when `max` is a valid value like 2. In this case the assertion would be something like:

`expect(console.error).toNotHaveBeenCalled()`